src/elements/02-elements/spinner/spinner.hbs

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" version="1.1" width="100px" height="100px" viewBox="0 0 50 50" class="rotatingspinner__svg">
+    <style>.rotatingspinner{overflow:hidden}.rotatingspinner__svg{animation:rotate 1.4s linear infinite;width:4rem;height:4rem}.rotatingspinner__svg--centered{display:block;margin:0 auto}.rotatingspinner__path{stroke:#1fab2e;stroke-linecap:round;stroke-dasharray:90,150;stroke-dashoffset:-35;animation:dash 1.4s ease-in-out infinite}@keyframes rotate{100%{transform:rotate(360deg)}}@keyframes dash{0%{stroke-dasharray:1,150;stroke-dashoffset:0}50%{stroke-dasharray:90,150;stroke-dashoffset:-35}100%{stroke-dasharray:90,150;stroke-dashoffset:-124}}</style>
```

Comment:
i am not sure about this.

- where is `.rotatingspinner` (the Block - **B**EM) used?
- Also why no **ys** prefix?
- we got a couple of classes in the <style> tag that are not used including `.rotatingspinner`, `.rotatingspinner__svg--centered`

Reply:
Thanks for feedback Razvan. I agree in all you bullets - and I will fix it.
The reason for this madness was that I saw this SVG file as a "Stand-alone" component with no external CSS or anything. But there is no reason not to change it to a "normal" component. I will do this and at the same time move the CSS out of the SVG/XML markup.

Reply:
Cleaned up code. Please look again @razvandk :)